templates/scripts/deleteuser.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+apt-get install pip
+pip install requests
+set -ex
+liquid_domain="${liquid_domain}"
```

Comment:
same as above: don't use any special Nomad or Jinja syntax.

In this bash script, I think it's best to simply use `$1` and `$2`.

The `${XXX}` syntax in Bash is problematic because it will be picked up by Jinja templating engine instead of Bash. `$1` and `$2`, which you can set to some named variables `$SOME_THING_1` and `$SOME_THING_2` do not have that problem.